Monika After Story/game/script-farewells.rpy

```diff
@@ -1451,4 +1451,29 @@ label bye_dinner_noon_to_mn:
     else:
         m 2euc "I guess it's dinner time for you."
         m 2esd "Well...{w=1}enjoy."
-    return
\ No newline at end of file
+    return
+    
+    init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_prompt_housework",
+            unlocked=True,
+            prompt="I'm going to do some housework.",
+            pool=True
+        ),
+        code="BYE"
+    )
+
+label bye_prompt_housework:
+    if mas_isMoniNormal(higher=True):
+            m 1eub "Doing your chores, [player]?"
+            m 1ekc "I would like to help you out with that, but there’s not much I can do about it since I’m stuck in here…“
```

Comment:
I assume you use Google docs or word to draft these out. We'd recommend you use a text editor or a code editor. RenPy doesn't see these quotation marks as valid syntax.

```suggestion
            m 1ekc "I would like to help you out with that, but there’s not much I can do about it since I’m stuck in here…"
```